manager/src/main/java/com/redhat/service/bridge/manager/api/user/BridgesAPI.java

```diff
@@ -66,16 +66,16 @@ public Response createBridge(BridgeRequest bridgeRequest) {
     }
 
     @GET
-    @Path("{bridgeId}")
-    public Response getBridge(@PathParam("bridgeId") @NotEmpty String bridgeId) {
-        Bridge bridge = bridgesService.getBridge(bridgeId, identityResolver.resolve(jwt));
+    @Path("{bridgeIdentifier}")
```

Comment:
@vaibhavjainwiz I think it's clearer for a user on how to use the API if this parameter is `idOrName`. The parameter is self explanatory.

Reply:
done